api/profile/profile.go

```diff
@@ -91,6 +91,10 @@ type Profile struct {
 	// AuthConnector (like "google", "passwordless").
 	// Equivalent to the --auth tsh flag.
 	AuthConnector string `yaml:"auth_connector,omitempty"`
+
+	// LoadAllHostCAs indicates that tsh should load the host CAs of all clusters
+	// instead of just the current cluster.
+	LoadAllHostCAs bool `yaml:"load_all_host_cas,omitempty"`
```

Comment:
```suggestion
	// LoadAllCAs indicates that tsh should load the CAs of all clusters
	// instead of just the current cluster.
	LoadAllCAs bool `yaml:"load_all_cas,omitempty"`
```